src/routes.rs

```diff
@@ -58,7 +59,11 @@ pub(crate) async fn get_invoice_impl(
         },
     };
 
-    // TODO: Check if the user has a Lightning address configured.
+    let metadata = state.client.metadata(zap_request.pubkey).await?;
```

Comment:
> Do you have a better idea on how to ensure we are not accepting bets from users without a lightning address.

Not doing it? :) 

> BTW. We do the same when we zap winners back.

That's slightly different because it doesn't effect the UX when zapping a note.